2016-contributing-to-your-career.md

```diff
@@ -0,0 +1,189 @@
+
+Chris DiBona, head of the Open Source Programs Office at Google, says:
+
+> Open source soft­ware is strate­gic to Google, and nat­u­rally we hire a great num­ber of open source devel­op­ers. Some­one who demon­strates their abil­ity by con­tribut­ing to open source projects shows that they are able to code in the real world in ways other devel­op­ers can not read­ily match. **_It’s the ulti­mate refer­ral._**
+
+According to the [2016 Open Source Jobs Report](www.linuxfoundation.org/news-media/announcements/2016/05/2016-open-source-jobs-report-companies-hungry-professional-open), 87% of hiring managers say it’s hard to find
+open source talent, and that "open source talent is one of the top priorities for recruitment this year." Basically, these companies want to throw money at you, but can’t find you.
+
+It turns out, there's a secret patented formula to getting any job you want:
+
+1. Learn skills and gain experience using those skills
+2. Create public artifacts that demonstrate your skills
+3. Make connections with people that need your skills and experience
+
+Open source is an incredibly effective way to gain experience, build a reputation, and make connections.
+
+## Becoming a contributor
+
+The first step to becoming a contributor is realizing that open source is made by people like you.
+
+> Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use.
+>
+> Once you learn that, you'll never be the same again.
+>
+> – Steve Jobs
+
+Seriously, think about this for a minute. Look around you right now. Pick up anything that is in reach and hold it in your hand. The display you're reading this on, whatever you're sitting on, the clothes you're wearing, the building that you're in. These were all made my people just like you. Someone first had the thought of making this thing. Maybe they sketched it out on paper, or told a friend about it. Maybe they put together a presentation and gave it to their team or their boss. Someone else saw that idea and offered feedback or a suggestion to improve it. Through hard work over time, that thing evolved into whatever it is today, thanks to people just like you.
+
+[![Despite it being imaginary, I already have SUCH a strong opinion on the cord-switch firing incident.](https://imgs.xkcd.com/comics/work.png)](https://xkcd.com/1741/)
+
+Open source software is like that, except that _anyone_ can contribute. Some contributors are students, some are hobbyists, some work for startups, some work for large companies, some are technical prodigies, and most are just normal people like you and me.
+
+Just remember, **It's ok to be new**. Every piece of open source software you use is made by people that we're once in your situation. Over time you'll likely suffer from "imposter syndrome". Unfortunately, that feeling doesn't go away. 
```

Comment:
I'm a little bit torn about the reference to impostor syndrome. The feeling that you are not qualified is generally a pretty legitimate feeling when you're new. There are legit a bunch of things you don't know yet. Impostor Syndrome is about not being able to internalize accomplishments, not being afraid that you're not good enough because you're new.

Ref:
- "You don't have impostor syndrome" [blog post](https://medium.com/@aliciatweet/impostor-syndrome-is-not-just-a-confidence-problem-dea670e59f6e)
- "Impostor Syndrome is not just a confidence problem" [blog post](https://medium.com/@aliciatweet/impostor-syndrome-is-not-just-a-confidence-problem-dea670e59f6e)